build.xml

```diff
@@ -32,6 +32,9 @@
   </path>
 
   <path id="test.classpath">
+    <fileset dir="${third_party.dir}">
+        <include name="guava/*.jar"/>
```

Comment:
where are these coming from? they're not in this CL?

Reply:
Took me a little while to figure that out as well. The "tests", such as they were, did not exercise enough prod code to hit the prod dependency on Guava. We could instead include gsa.build.classpath here, which might be slightly more conventional.

Reply:
I was mistaken about this. The initial commit was using Strings.isNullOrEmpty, but that's been removed, and this can go away. The only use of Guava in gsafeed is an `@VisibleForTesting` in the copied GsaFeedFileSender.